packages/@ember/-internals/runtime/lib/system/array_proxy.js

```diff
@@ -111,11 +111,11 @@ export default class ArrayProxy extends EmberObject {
       this._arrangedContentRevision = value(this._arrangedContentTag);
     }
 
-    this._addArrangedContentArrayObsever();
```

Comment:
LOL @ this typo, I stared at this line for ~ 3 minutes trying to figure out why it was in the git diff 😝